### ISSUE TITLE
Scarab dont detect Hollow Knight .exe and i cant download any mod

### ISSUE THREAD

**Issue by Jlwht** [fifty-six/Scarab#210]
### Scarab Version

v2.5.0.0

### Hollow Knight Version

v1.5.78.11833

### Steps to Reproduce

Tell me what to do

### Expected Behaviour

Solve my problem

### Actual Behaviour

When i try to download a mod  the next text appears:
"System.IO.FileNotFoundException: Could not find file: 'C:Program Files (x86)\Steam\steamapss\common\Hollow Knight\hollow_knight_Data\Managed\Assembly-CSharp.dll'.

When i try to select .exe this appears:
![image](https://github.com/fifty-six/Scarab/assets/161541898/b5490262-3b09-42c2-bcf3-786b138f629f)

### Logs

2024-02-27 20:19:12.323 -05:00 [INF] Launching...
2024-02-27 20:19:12.883 -05:00 [INF] Initializing to normal mode (.cctor)
2024-02-27 20:19:14.613 -05:00 [INF] Checking if up to date...
2024-02-27 20:19:14.619 -05:00 [INF] Current version of installer is 2.5.0.0
2024-02-27 20:19:15.215 -05:00 [INF] Loading settings.
2024-02-27 20:19:15.233 -05:00 [INF] Trying path C:\Program Files (x86)/Steam/steamapps/common/Hollow Knight\Hollow Knight_Data/Managed
2024-02-27 20:19:15.233 -05:00 [INF] Trying path C:\Program Files (x86)/Steam/steamapps/common/Hollow Knight\hollow_knight_Data/Managed
2024-02-27 20:19:15.234 -05:00 [INF] Selected path root C:\Program Files (x86)/Steam/steamapps/common/Hollow Knight with suffix hollow_knight_Data/Managed was missing Assembly-CSharp.dll!
2024-02-27 20:19:15.241 -05:00 [INF] Trying path C:\\Program Files (x86)\\Steam\steamapps\common\Hollow Knight\Hollow Knight_Data/Managed
2024-02-27 20:19:15.241 -05:00 [INF] Trying path C:\\Program Files (x86)\\Steam\steamapps\common\Hollow Knight\hollow_knight_Data/Managed
2024-02-27 20:19:15.241 -05:00 [INF] Selected path root C:\\Program Files (x86)\\Steam\steamapps\common\Hollow Knight with suffix hollow_knight_Data/Managed was missing Assembly-CSharp.dll!
2024-02-27 20:19:15.242 -05:00 [INF] Unable to detect installation path for settings, selecting manually.

**Comment by fifty-six** [comment]
Is Assembly-CSharp.dll in the path it's looking at if you go check in file explorer? Have you tried verifying your game integrity before running it?

**Comment by goldjojo** [comment]
I was in the middle of typing a reply as I figured out the solution for fixing what my similar problem was.

I had the same issue. every time I tried to download a mod or re-install the API, the "Assembly-CSharp.dll" file changed to "Assembly-CSharp.dll.v" (.v at the end)

renaming the file back to Assembly-CSharp.dll, setting the path again, and reopening the launcher fixed the issue

https://github.com/fifty-six/Scarab/assets/46782357/09a0e2f6-73d2-4b75-904a-f025a1b66144

**Comment by sgomez-2020088** [comment]
Yeah it solves de mod trouble but now i can't open de game while using the mods, bruh
